lapack/testlapack/dlaexc.go

```diff
@@ -129,77 +149,21 @@ func testDlaexc(t *testing.T, impl Dlaexcer, wantq bool, n, j1, n1, n2, extra in
 			}
 			diff := tmat.Data[i*tmat.Stride+j] - tmatCopy.Data[i*tmatCopy.Stride+j]
 			if diff != 0 {
-				t.Errorf("%v: unexpected modification of T[%v,%v]", prefix, i, j)
+				t.Errorf("%v: unexpected modification of T[%v,%v]", name, i, j)
 			}
 		}
 	}
 
-	if n1 == 1 {
```

Comment:
The alternative to deleting these blocks would be to increase tolerance to `1e-7` or to have a separate tolerance only for these eigenvalue checks. Since I already check that T is Schur canonical, that it is similar to the original T and that it is not modified outside of the row/column band, these checks are in principle redundant and removing them also simplifies the test.